pkgs/development/libraries/pcre2/default.nix

```diff
@@ -3,7 +3,8 @@
   stdenv,
   fetchurl,
   updateAutotoolsGnuConfigScriptsHook,
-  withJitSealloc ? true,
+  # Causes consistent segfaults on ELFv1 PPC64 when trying to use Perl regex in gnugrep
+  withJitSealloc ? !(stdenv.hostPlatform.isPower64 && stdenv.hostPlatform.isAbiElfv1),
```

Comment:
This one been reported?

Reply:
Prolly not with any details, so I'll try to do some builds with debug symbols and get a proper trace of the segfault.

Though searches online and upstream issues & comments suggest that this allocator has known-issues that lead to segfaults like this (like breaking across `fork()`s, which is why our PHP packages set and OpenSUSE in general is not enabling it), and it's explicitly marked as experimental… Iunno if we should be enabling this in general, but I didn't want to make a decision like that in this PR.

Reply:
Yeah, that makes sense. I think this is fine but would be nice to report it still.

Enabling it is at least justified with "fix pcre jit in systemd units that set MemoryDenyWriteExecute=true like gitea", which does sound desirable…

Reply:
FWIW, it is not only documented as experimental AND explained to have several bugs to help discourage people, but specifically to make any user clear that it is NEVER intended to be enabled unless it is used in a very controlled setup (therefore in a library that was compiled statically and linked int a binary that wouldn't trigger them).

there is no justification for building this in a library that is linked dynamically and used broadly for an reason, specially if it is to try to workaround a specific control like MemoryDenyWriteExecute=true, because it doesn't even do that.

it is a hack, AND is still vulnerable to the same issue that denying pages that are writeable and executable was trying to prevent.